test/image/mocks/multiple_axes_anchored.json

```diff
@@ -92,39 +92,51 @@
   ],
   "layout": {
     "title": "multiple y-axes example",
-    "width": 800,
-    "height": 500,
+    "width": 1200,
+    "height": 800,
+    "legend": {
+        "orientation": "h",
+        "y": 0,
+        "yanchor": "top"
+    },
     "xaxis": {
       "anchoroffset": true,
-      "title": "This needs to be here for now",
-      "domain": [
-        0.2,
-        0.8
-      ]
+      "title": "This needs to be here for now"
```

Comment:
Sure, though this is just the testing template json for the plotly framework, not default titles.